CMakeLists.txt

```diff
@@ -45,9 +45,15 @@ endif(UNIX)
 
 if(APPLE)
     add_definitions(-DHS_BUILD_FOR_APPLE)
+    set(CMAKE_XCODE_GENERATE_SCHEME FALSE)
```

Comment:
This overrides any use setting, which is bad practice in general. Let's set the cache instead.

Reply:
So that would be this instead?

`set(CMAKE_XCODE_GENERATE_SCHEME FALSE CACHE BOOL)`

Reply:
`set(CMAKE_XCODE_GENERATE_SCHEME FALSE CACHE BOOL "")`

CMake gets mad if you leave out the docstring :(